indico/web/views.py

```diff
@@ -219,14 +209,18 @@ def display(self, **params):
                                        multi_value_types=list)
         custom_js = self._asset_env['custom_js'].urls() if 'custom_js' in self._asset_env else []
         custom_css = self._asset_env['custom_sass'].urls() if 'custom_sass' in self._asset_env else []
-        css_files = map(self._fix_path, self.getCSSFiles() + plugin_css + self.get_extra_css_files() + custom_css)
-        print_css_files = map(self._fix_path, self.getPrintCSSFiles())
-        js_files = map(self._fix_path, self.getJSFiles() + plugin_js + custom_js)
+        css_files = map(self._fix_path, plugin_css + self.get_extra_css_files() + custom_css)
+        js_files = map(self._fix_path, plugin_js + custom_js)
 
         body = to_unicode(self._display(params))
+        webpack = current_app.extensions['flask-webpackext']
+
+        bundles = map(lambda x: webpack.manifest[x], self.bundles)
```

Comment:
list comprehension? ;)

Reply:
I don't even need a list. A generator is enough 😉